src/core_plugins/kibana/public/context/query_parameters/actions.js

```diff
@@ -43,7 +46,13 @@ export function QueryParameterActionsProvider() {
     )
   );
 
+  const addTermsFilter = (state) => (field, values, operation) => {
+    const { queryParameters: { indexPatternId } } = state;
+    filterManager.add(field, values, operation, indexPatternId);
```

Comment:
I'm not sure how useful it would be to influence the ordering of the fields in the Discover field chooser via some action in the Context view. It might actually cause more confusion. :thinking: 

@tbragin what is your opinion?